index.html

```diff
@@ -41,67 +41,92 @@
-        <h3>More Details</h3>
+        <h3>Main Program</h3>
         <div class="line"></div>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nam unde iusto vel aperiam consequatur ea, alias a corrupti aliquid odio voluptatem, nemo, cumque ad in. Eum quaerat modi vero nemo.</p>
-    </div>
-    <img class="details-img" src="images/details.jpg" alt="">
+      </div>
+      <div class="main-program-content parent">
+        <a href="" class="child"><i class="fab fa-angellist"></i></a>
+        <h3>Chemistry</h3>
+        <p class="child">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem fugit porro vel dolor. Dolore, expedita.</p>
+      </div>
+      <div class="main-program-content parent">
+        <a href="" class="child"><i class="fab fa-angellist"></i></a>
+        <h3>Physics
+        </h3>
+        <p class="child">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum amet reprehenderit saepe porro qui totam!</p>
+      </div>
+      <div class="main-program-content parent">
+        <a href="" class="child"><i class="fab fa-angellist"></i></a>
+        <h3>Math</h3>
+        <p class="child">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Numquam, culpa totam. Velit quod vel sunt?</p>
+      </div>
+      <div class="main-program-content parent">
+        <a href="" class="child"><i class="fab fa-angellist"></i></a>
+        <h3>English</h3>
+        <p class="child">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Numquam, culpa totam. Velit quod vel sunt?</p>
+      </div>
+      <div class="main-program-content parent">
+        <a href="" class="child"><i class="fab fa-angellist"></i></a>
+        <h3>Ict</h3>
+        <p class="child">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nobis ullam, facilis vel optio officia quia.</p>
+      </div>
+      <p class="clickLink">SEE THE WHOLE PROGRAM</p>
+      <button class="btn">Join Us</button>
     </section>
-    <section class="event">
-        <div class="underline-content">
-            <h3>Some Previous Events</h3>
-            <div class="line"></div>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem ut hic fuga itaque qui saepe necessitatibus vitae sint, illum expedita.</p>
-        </div>
-        <div class="event-info">
-            <div class="event-shade">
-                <img class="event-img" src="images/event1.jpg" alt="">
-                <div class="event-date">
-                  <h3>2013</h3>
-                  <p>Lorem ipsum dolor sit amet consectetur.</p>
-                </div>
-            </div>
-            <div class="event-shade">
-              <img class="event-img" src="images/event2.jpg" alt="">
-              <div class="event-date">
-                <h3>2015</h3>
-                <p>Lorem ipsum dolor sit amet consectetur.</p>
-              </div>
-          </div>
-        </div>
+    <section class="featured-speaker"></section>
```

Comment:
- [ ] Great job creating the speaker's section dynamically using JavaScript, it looks good and is well-aligned. It will be great to add the extra styling behind each speaker image so as to follow the required design.
- Your design:
![c5](https://user-images.githubusercontent.com/85113567/152383553-43f37fb0-a1d2-4cd5-85cc-416e8648fad3.PNG)
- Required design:
![c6](https://user-images.githubusercontent.com/85113567/152383606-030e9567-eeea-467f-aecf-9c98adb154de.PNG)